pkl-core/src/main/resources/org/pkl/core/errorMessages.properties

```
@@ -1126,3 +1130,45 @@ External {0} reader does not support scheme `{1}`.
 
 externalReaderAlreadyTerminated=\
 External reader process has already terminated.
+
+keywordNotAllowedHere=\
+Keyword `{0}` is not allowed here. (If you must use this name as identifier, enclose it in backticks.)
+
+unexpectedEndOfFile=\
+Unexpected end of file.
+
+wrongHeaders=\
+{0} cannot have doc comments, annotations or modifiers.
+
+invalidTopLevelToken=\
+Invalid token at position. Expected class, typealias, method or property.
+
+interpolationInConstant=\
+String constant cannot have interpolated values.
+
+unexpectedToken=\
+Unexpected token. Expected `{0}`.
+
+unexpectedToken2=\
+Unexpected token. Expected `{0}` or `{1}`.
+
+unexpectedTokenMany=\
+Unexpected token. Expected one of {0}.
+
+unexpectedTokenForExpression=\
+Unexpected token.
+
+unexpectedTokenForType=\
+Unexpected token. Expected a type.
+
+typeAnnotationInAmends=\
+Expected expression but got object body. Amended properties cannot have type annotations.
```

Missing snippet test.

Also, perhaps:

```suggestion
Properties with type annotations cannot have object bodies.

To define both a type annotation and an object body, try using assignment instead.
For example: `obj: Bird = new { ... }`.
```